### PR TITLE
Add a workaround to link to Java Code

### PR DESCRIPTION
**Pull request by TobiasNx**
Since all fixes are in one of three Java Classes I created a workaround to link to the Java Code using the GitHub search.

@maipet could you have a look if this meets your demands?
@blackwinter could you have a look if this is okay for you?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,8 @@ Options:
 include("<path>"[, <dynamicLocalVariables>...])
 ```
 
+[Java code](https://github.com/search?q=repo%3Ametafacture%2Fmetafacture-fix+path%3A%2F%5Emetafix%5C%2Fsrc%5C%2Fmain%5C%2Fjava%5C%2Forg%5C%2Fmetafacture%5C%2Fmetafix%5C%2FFixMethod.java%2F+include&type=code)
```
Comment (blackwinter) [inline README.md:169]: 1. Inconsistent capitalization (only here).
2. Exact matching (best to tighten everywhere).
3. Simpler path qualifier (file names are unique).

```suggestion
[Java Code](https://github.com/search?q=repo:metafacture/metafacture-fix+path:FixMethod.java+"+include+{"&type=code)
```

**Review by blackwinter — CHANGES_REQUESTED** [review]
No objection to the idea itself, but I think the links could be improved.

File: README.md (hunk 1 above)
Comment (TobiasNx) [reply README.md:169]: +1 I will use this everywhere

**Review by blackwinter — APPROVED** [review]
:+1:

**Comment by TobiasNx** [comment]
@maipet said that the need for double clicking to get to the code is confusing.
Unfortunatly github does not provide a search that links abstractly to a java class/method and I provided a workaround.

We will test and see if the workaround is sufficient for now.

**Pull request merged**